release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -1014,6 +1014,29 @@ The root cause of this issue is unknown and no workaround currently exists.
 +
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1913279[*BZ#1913279*])
 
+* UPI installation on OpenStack with Kuryr times out while waiting for bootstrapping:
++
+[source,terminal]
+----
+INFO Waiting up to 20m0s for the Kubernetes API at https://api.ostest.shiftstack.com:6443...
+INFO API v1.20.0+ba45583 up
+INFO Waiting up to 30m0s for bootstrapping to complete...
+ERROR Attempted to gather ClusterOperator status after wait failure: listing ClusterOperator objects: Get "https://api.ostest.shiftstack.com:6443/apis/config.openshift.io/v1/clusteroperators": dial tcp 10.46.44.166:6443: connect: connection refused
+INFO Use the following commands to gather logs from the cluster
+INFO openshift-install gather bootstrap --help
+FATAL failed to wait for bootstrapping to complete: timed out waiting for the condition
+----
++
+This is caused by changes in how Kuryr detects the Neutron subnet of the {product-title} nodes. The workaround is to not remove the master Machine manifests, so in step 2 of "Creating the Kubernetes manifest and Ignition config files" in installation instructions the command to use becomes:
++
+[source,terminal]
+----
+$ rm -f openshift/99_openshift-cluster-api_worker-machineset-*.yaml
+----
++
+The issue is scheduled to be fixed in one of the z-stream {product-title} 4.7 updates.
++
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1927244[*BZ#1927244*])
```

Comment:
@dulek So, I'm basically thinking this:

```suggestion
* Installing a cluster on {rh-openstack} that uses Kuryr times out with the following messages during bootstrapping: 
+
[source,terminal]
----
INFO Waiting up to 20m0s for the Kubernetes API at https://api.ostest.shiftstack.com:6443...
INFO API v1.20.0+ba45583 up
INFO Waiting up to 30m0s for bootstrapping to complete...
ERROR Attempted to gather ClusterOperator status after wait failure: listing ClusterOperator objects: Get "https://api.ostest.shiftstack.com:6443/apis/config.openshift.io/v1/clusteroperators": dial tcp 10.46.44.166:6443: connect: connection refused
INFO Use the following commands to gather logs from the cluster
INFO openshift-install gather bootstrap --help
FATAL failed to wait for bootstrapping to complete: timed out waiting for the condition
----

The time out is caused by changes in how Kuryr detects the {rh-openstack} Networking service (neutron) subnet of the cluster's nodes. 

As a workaround, do not remove the control plane machine manifests as described by "Creating the Kubernetes manifest and Ignition config files" in the installation documentation.When you are instructed to run the following command:
+
[source,terminal]
----
$ rm -f openshift/99_openshift-cluster-api_master-machines-*.yaml openshift/99_openshift-cluster-api_worker-machineset-*.yaml
----

Run this command instead:
+
[source,terminal]
----
$ rm -f openshift/99_openshift-cluster-api_worker-machineset-*.yaml
----
```
I could see about adjusting the prod Kuryr UPI docs to use the latter command, instead. I could then revert the change after the bug is fixed. This would trigger a QE review, though, and it's not as easy to get approved after freeze as a release note is. 

Thoughts?

Reply:
I'd rather not touch the documentation itself. The step with removing master machines manifests is correct and in the future not removing them may have adverse side effects. I don't want anyone to write this workaround into their scripts without intent.

Reply:
Works for me. Just wanted to present options. :)